libs/blocks/section-metadata/section-metadata.css

```diff
@@ -26,7 +26,7 @@
 }
 
 .section.l-spacing {
-  padding: var(--spacing-l) 0;
+  padding: var(--spacing-xl) 0;
```

Comment:
Everywhere there is a mismatch of spacing (or really any vars) should be fixed. A platform should be consistent. We should not have a class name as "L" and the padding be "XL". Some of the sizing is weird... like a Marquee large will have padding of spacing--xxl (or whatever), but in these direct uses, they should be the same.

Reply:
Copy that, This is fixed. Was an oversight not looking closely at the util class. my bad